manifests/2.5.0/opensearch-2.5.0.yml

```diff
@@ -155,3 +155,12 @@ components:
       - windows
     checks:
       - gradle:properties:version
+  - name: security-analytics
+    repository: https://github.com/opensearch-project/security-analytics.git
+    ref: 2.x
+    platforms:
+      - linux
+      - windows
+    checks:
+      - gradle:properties:version
+      - gradle:dependencies:opensearch.version
```

Comment:
Looking at the past manifests, looks like security-analytics has only 1 check. https://github.com/opensearch-project/opensearch-build/blob/main/manifests/2.4.0/opensearch-2.4.0.yml#L161-L167

Reply:
thanks @gaiksaya . fixed it